sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Customizations/QuickPulseSDKClientAPIsRestClient.cs

```diff
@@ -45,10 +46,11 @@ internal partial class QuickPulseSDKClientAPIsRestClient
                         CollectionConfigurationInfo value = default;
                         if (message.Response.Headers.ContentLength != 0)
                         {
+                            // TODO: This deserialization was auto generated. I assume this is used for filtering.
                             using var document = JsonDocument.Parse(message.Response.ContentStream);
                             value = CollectionConfigurationInfo.DeserializeCollectionConfigurationInfo(document.RootElement);
                         }
-                        return ResponseWithHeaders.FromValue<object, QuickPulseSDKClientAPIsPingHeaders>(value, headers, message.Response);
+                        return new QuickPulseResponse(success: true, message.Response.Headers);
```

Comment:
I really want to avoid boxing the return objects. 
Would you be okay with `out` params here? The reason I didn't is that it already has 10 parameters.

Reply:
discussed offline. previous implementation was returning a new class for every ping/post. new implementation returns a readonly struct, so this is an improvement.